metrics-and-logs/exporting-logs.html.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: Exporting Logs
-layout: framework_docs
+title: Export logs
+layout: docs
 objective: Aggregate your logs to a service of your choice.
-order: 1
-status: beta
+nav: firecracker
+redirect_from: /docs/going-to-production/monitoring/exporting-logs/
 ---
 
-Your applications' output to `stdout` become logs in Fly.io. The logs for all VM instances within any given app can be viewed within an app's Monitoring page. However, these logs are intermingled and do not persist very long.
+The Fly Log Shipper app enables you to aggregate your app's logs to a service of your choice.
 
-If you need or want your logs to be aggregated to one spot, more organized, or to persist longer than we keep them, you can!
+Your app's output to `stdout` become logs in Fly.io. The logs for all Machines in an app can be viewed within the app's dashboard. However, we only persist these logs for about two days. If you need or want your logs to be aggregated to one spot, be more organized, or to persist longer than we keep them, then you can do that with the Fly Log Shipper.
```

Comment:
So this is misleading. You can only 'live tail' logs in the dashboard - same as when using `flyctl`. We do persist logs for 2 days in ElasticSearch, but that's for internal use only. The new Grafana log search persists for 30 days, but retention will be dropped once we introduce paid log plans.

This whole section might simply say "Live log tail and log search are good enough for most things. But if you need to export your logs to an external service, such as Datadog or AWS S3, you can use the Log Shipper."